runtime/compiler/runtime/JITServerAOTCache.cpp

```diff
@@ -792,17 +795,92 @@ JITServerAOTCacheMap::~JITServerAOTCacheMap()
    TR::Monitor::destroy(_monitor);
    }
 
+std::string
+JITServerAOTCacheMap::buildCacheFileName(const std::string &cacheDir, const std::string &cacheName)
+   {
+   std::string cacheFileName;
+   // If a directory is given, add it in the front of the cache name
+   if (!cacheDir.empty())
+      cacheFileName = cacheDir + "/"; // TODO: is there something more portable?
+   return cacheFileName + "AOTCache." + cacheName + ".bin";
```

Comment:
".bin" sounds like a Windows thing to me. Also maybe we should include "JITServer" in the name.